tests/syft/core/store/memory_storage_test.py

```diff
@@ -1,39 +1,106 @@
-# import numpy as np
-
-# from syft.core.store.storeable_object import StorableObject
-# from syft.core.store.store_memory import MemoryStore
-# from syft.core.common import UID
-
-
-# def test_create_memory_storage():
-#     store = MemoryStore()
-#
-#     k1 = UID()
-#     data1 = StorableObject(
-#         id=k1,
-#         data=np.array([1, 2, 3, 4]),
-#         description="This is a dummy test",
-#         tags=["dummy", "test"],
-#     )
-#
-#     store[k1] = data1
-#     assert k1 in store
-#     assert len(store) == 1
-#
-#     k2 = UID()
-#     data2 = StorableObject(
-#         id=k1,
-#         data=np.array([1, 2, 3, 4]),
-#         description="This is a dummy test",
-#         tags=["dummy", "test"],
-#     )
-#     store[k2] = data2
-#
-#     assert store.get_objects_of_type(np.array) == [data1, data2]
-#     assert list(store.keys()) == [k1, k2]
-#     assert list(store.values()) == [data1, data2]
-#
-#     store.clear()
-#
-#     assert len(store) == 0
-#     assert store.__sizeof__() == 48
+import torch as th
+import re
+
+from syft.core.store.storeable_object import StorableObject
+from syft.core.store.store_memory import MemoryStore
+from syft.core.common import UID
+
+from typing import Tuple, List
+
+
+def generate_id_obj(data: th.Tensor, description: str, tags: List[str]) -> \
+        Tuple[UID, StorableObject]:
+    id = UID()
+    obj = StorableObject(
+        id=id,
+        data=data,
+        description=description,
+        tags=tags
+    )
+
+    return id, obj
+
+
+def test_create_memory_storage() -> None:
+    store = MemoryStore()
+
+
+def test_set_item() -> None:
+    store = MemoryStore()
+    id1, obj1 = generate_id_obj(data=th.Tensor([1, 2, 3, 4]),
+                                description="Dummy tensor",
+                                tags=["dummy", "tensor"])
+
+    store[id1] = obj1
+    assert id1 in store
+    assert len(store) == 1
+
+
+def test_store_item() -> None:
+    store = MemoryStore()
+    id1, obj1 = generate_id_obj(data=th.Tensor([1, 2, 3, 4]),
+                                description="Dummy tensor",
+                                tags=["dummy", "tensor"])
+
+    store.store(obj=obj1)
+    assert id1 in store
+    assert len(store) == 1
+
+
+def test_get_objects_of_type() -> None:
+    store = MemoryStore()
+    id1, obj1 = generate_id_obj(data=th.Tensor([1, 2, 3, 4]),
+                                description="Dummy tensor",
+                                tags=["dummy", "tensor"])
+    id2, obj2 = generate_id_obj(data=th.Tensor([1, 2, 3]),
+                                description="Another dummy tensor",
+                                tags=["another", "dummy", "tensor"])
+
+    store[id1] = obj1
+    store[id2] = obj2
+    assert store.get_objects_of_type(th.Tensor) == [obj1, obj2]
```

Comment:
Could we add objects of another type and test the method with another argument to be sure that it really works as intended?

Reply:
Yes, working on it.

Reply:
Done.